config/development.yaml

```diff
@@ -36,6 +36,10 @@ scenarios:
         writers:
           replicas: 10
           throughput: 100
+          args:
+            - url: #pushURL
+            - logps: #throughput
+            - tenant: #tenantID
```

Comment:
For `throughput` we can certainly eliminate `writers.throughput` as a field and replace it directly with `args.logps: 100`

There is no need to configure the `pushURL`. This is auto-generated by `benchCfg.Loki.PushURL()`. Since we have two load client we can standardize to provide a cli flag `--url`. Same goes for `--tenant`.

In general I would like to keep out placeholders like `#pushURL` here. The idea for `args` was to amend/extend the logger configuration with extra args, e.g. imagine cluster-logging-load-client provides `--throughput` instead of `--logps`.

Reply:
Fixed
DONE